api/server/router/container/container.go

```diff
@@ -60,6 +60,7 @@ func (r *containerRouter) initRoutes() {
 		// PUT
 		local.NewPutRoute("/containers/{name:.*}/archive", r.putContainersArchive),
 		// DELETE
+		local.NewDeleteRoute("/containers/_all", r.deleteAllContainers),
```

Comment:
@MHBauer Tibor and I actually had a discussion about it on IRC today. "RESTful" is not an official specification, so there's no standard to point to (apart from conventions). See our discussion here: https://botbot.me/freenode/docker-maintainers/2015-12-07/?msg=55595540&page=1

I agree that a trailing slash for a "collection" could be nice, but relying on it for different behavior sounded tricky to me (I'm worried that client frameworks may add a trailing slash (or not), and (for example) a proxy-server redirecting to URL's with slash).